### DISCUSSION TITLE
Different search results in Browser and e.g. Postman

### DISCUSSION THREAD

**Discussion by jeftadlvw** [komoot/photon#811] [Q&A] [answered]
I might have missed something on the website, README or existing discussions, but I'm getting different search results when running the same query in the browser or a 3rd party application like Postman.

E.g., when executing a search query for my college,
```
https://photon.komoot.io/api?q=westfälische+hochschule+campus+bocholt
```

I get this result in the browser (which is correct):
```json
{
  "features": [
    {
      "geometry": {
        "coordinates": [
          6.65235603579384,
          51.8396047
        ],
        "type": "Point"
      },
      "type": "Feature",
      "properties": {
        "osm_id": 25144486,
        "extent": [
          6.6497734,
          51.8408903,
          6.6550292,
          51.8387626
        ],
        "country": "Deutschland",
        "city": "Bocholt",
        "countrycode": "DE",
        "postcode": "46397",
        "county": "Kreis Borken",
        "type": "house",
        "osm_type": "W",
        "osm_key": "amenity",
        "housenumber": "265",
        "street": "Münsterstraße",
        "district": "Bocholt",
        "osm_value": "college",
        "name": "Westfälische Hochschule (Campus Bocholt)",
        "state": "Nordrhein-Westfalen"
      }
    }
  ],
  "type": "FeatureCollection"
}
```

But in Postman, I get this response (I get the busstop, but not the college itself):
```json
{
    "features": [
        {
            "geometry": {
                "coordinates": [
                    6.6484331,
                    51.8395206
                ],
                "type": "Point"
            },
            "type": "Feature",
            "properties": {
                "osm_id": 5347703580,
                "country": "Deutschland",
                "city": "Bocholt",
                "countrycode": "DE",
                "postcode": "46397",
                "county": "Kreis Borken",
                "type": "house",
                "osm_type": "N",
                "osm_key": "highway",
                "street": "Leipziger Straße",
                "district": "Bocholt",
                "osm_value": "bus_stop",
                "name": "Westfälische Hochschule",
                "state": "Nordrhein-Westfalen"
            }
        }
    ],
    "type": "FeatureCollection"
}
```

How does this happen and how can I configure my 3rd party application to get the same results as the browser?

**Comment by mtmail** [comment, marked as answer]
Do some testing with the `lang` Photon parameter. Browsers send a `Accept-Language` HTTP header (https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Language) by default and Photon is probably looking at that if no `lang` parameter is set in a request.

**Reply by jeftadlvw** [reply]
That's it! Thank you!
Maybe this should be documented in a small sentence? I could open a pull request if interested.

Purely out of curiosity: What's the default language/behaviour if no language is passed to Photon (through headers or url params)?

**Reply by mtmail** [reply]
If both unset then the local name is used. You can test that searching for the country Switzerland https://www.openstreetmap.org/relation/51701 The local name here is "Schweiz/Suisse/Svizzera/Svizra". The local name of Japan is 日本 etc.

**Reply by karussell** [reply]
> Maybe this should be documented in a small sentence? I could open a pull request if interested.

IMO this sounds good. Maybe one sentence in the readme under "Search API"?

**Reply by mtmail** [reply]
@karussell https://github.com/komoot/photon/pull/812

**Reply by karussell** [reply]
Ups, didn't made the connection. Thanks :)
